README.md

```diff
@@ -0,0 +1,43 @@
+## Edit Discourse posts locally
+Discedit allows you to edit [Discourse](https://www.discourse.org/) posts in your favourite local text editor. It woks by pulling a post from Discourse, openning it in your favourite text editor and automatically pushing the edits. 
```

Comment:
Typos: woks, openning

Also, let's please keep the tool name in lowercase at all times: "The discedit tool ...", for example.

Also, let's please use "topics" and not "posts". We can eventually support individual posts, but that's not the case just now as the main requirement was editing of topics.